packages/cactus-plugin-ledger-connector-ethereum/src/test/typescript/integration/geth-contract-deploy-and-invoke-using-json-object-v1.test.ts

```diff
@@ -227,22 +227,25 @@ describe("Ethereum contract deploy and invoke using keychain tests", () => {
   });
 
   test("deployContract without contractJSON should fail", async () => {
-    try {
-      await apiClient.deployContract({
+    await expect(
+      apiClient.deployContract({
         contract: {} as ContractJsonDefinition,
         web3SigningCredential: {
           ethAccount: WHALE_ACCOUNT_ADDRESS,
           secret: "",
           type: Web3SigningCredentialType.GethKeychainPassword,
         },
-      });
-      fail("Expected deployContract call to fail but it succeeded.");
-    } catch (error) {
-      console.log("deployContract failed as expected");
-    }
+      })
+    ).rejects.toThrow();
+    
+    console.log("deployContract failed as expected");
+    
   });
 
   test("deployContract with additional parameters should fail", async () => {
+
+    //this try-catch statement was not refactored because calling deployContract with additional parameters is actually not
```

Comment:
@ashnashahgrover Good catch! In that case let's assert success: remove the try-catch and assert that the promise resolves. It makes it much less confusing.

Reply:
But isn't that condition of the test that "deployContract with additional parameters should fail" ?

Reply:
Have made this issue:

https://github.com/hyperledger/cacti/issues/3487

To resolve this instead.